tools/benchmark/cmd/put.go

```diff
@@ -150,3 +160,40 @@ func max(n1, n2 int64) int64 {
 	}
 	return n2
 }
+
+func hashKV(cmd *cobra.Command, clients []*v3.Client) {
+	var epHashKVRev int64
+	eps, err := cmd.Flags().GetStringSlice("endpoints")
+	if err != nil {
+		panic(err)
+	}
+	for i, ip := range eps {
+		eps[i] = strings.TrimSpace(ip)
+	}
+	st := time.Now()
+	var results string
+	ctx, cancel := context.WithCancel(context.Background())
+	clients[0].HashKV(ctx, eps[0], epHashKVRev)
+	host := eps[0]
+
+	resphash, errhash := clients[0].HashKV(ctx, host, epHashKVRev)
+	respstatus, errstatus := clients[0].Status(ctx, host)
+	cancel()
+	if errhash != nil {
+		fmt.Fprintf(os.Stderr, "Failed to get the hashkv of endpoint %s (%v)\n", host, errhash)
+		panic(err)
+	}
+	results += fmt.Sprintf("\nHaskKV Summary:\n")
+	results += fmt.Sprintf("  HashKV: %v \t\n", resphash.Hash)
+	results += fmt.Sprintf("  Endpoint: %v \t\n", host)
+	results += fmt.Sprintf("  Time taken to get hashkv: %v \t\n", time.Since(st).String())
+
+	// Get the db size
+	if errstatus != nil {
+		fmt.Println(fmt.Fprintf(os.Stderr, "Failed to get the status of endpoint %s (%v)\n", host, errstatus))
+		panic(err)
+	}
+	dbsize := float64(respstatus.DbSize) / (1000 * 1000 * 1000)
+	results += fmt.Sprintf("  DB size: %vGB \t\n", strconv.FormatFloat(dbsize, 'f', 2, 64))
+	fmt.Println(results)
```

Comment:
@spzala Let's use `humanize.Bytes` to string-ify db size. Then, this should be good to merge. Thanks!

Reply:
@gyuho thanks, this's good to know. Will make changes.